internal/veracode-api-wrapper/updatemitigationinfo/scripts/updatemitigationinfo.py

```diff
@@ -0,0 +1,100 @@
+#  /*
```

Comment:
No need to have both # and /*. We can limit to just using # since this is python. Same applies to other lines below.

Same comment applies to `setup.py`